devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy

```diff
@@ -39,7 +39,7 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF", outClusters: "0019", manufacturer: "Megaman", model: "ZLL-ExtendedColor", deviceJoinName: "INGENIUM ZB RGBW Light"
 
 		// Innr
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "RB 185 C", deviceJoinName: "Innr Smart Bulb Color", mnmn:"SmartThings", "generic-rgbw-color-bulb-1800K-6500K"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "RB 185 C", deviceJoinName: "Innr Smart Bulb Color", mnmn:"SmartThings", vid: "generic-rgbw-color-bulb-1800K-6500K"
```

Comment:
I fixed this in `staging` yesterday but forgot to merge it into `master` until now. Can you rebase this branch on the latest master?

Reply:
This branch is too messed up. I've closed this one and created new PR